tests/regression/lp-1802581/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: Regression test for  1802581
```

Comment:
Stray space

Reply:
Could you please cherry pick the documentation of the issue from the snap attached to the bug report?

Reply:
Yep, the bugreport seems to be a private one.